commit dc91a19e6f71e1523f4ac179191a29b2131d74bb
Author: Joseph Myers <joseph@codesourcery.com>
Date:   Mon Jun 3 11:16:02 2019 +0000

    Add INADDR_ALLSNOOPERS_GROUP from Linux 5.1 to netinet/in.h.
    
    This patch adds INADDR_ALLSNOOPERS_GROUP from Linux 5.1 to
    netinet/in.h.
    
    Tested for x86_64.
    
            * inet/netinet/in.h (INADDR_ALLSNOOPERS_GROUP): New macro.

diff --git a/inet/netinet/in.h b/inet/netinet/in.h
index 03a31b634c8bfbed..c2d12a04aab6c022 100644
--- a/inet/netinet/in.h
+++ b/inet/netinet/in.h
@@ -204,6 +204,7 @@ enum
 #define INADDR_UNSPEC_GROUP	((in_addr_t) 0xe0000000) /* 224.0.0.0 */
 #define INADDR_ALLHOSTS_GROUP	((in_addr_t) 0xe0000001) /* 224.0.0.1 */
 #define INADDR_ALLRTRS_GROUP    ((in_addr_t) 0xe0000002) /* 224.0.0.2 */
+#define INADDR_ALLSNOOPERS_GROUP ((in_addr_t) 0xe000006a) /* 224.0.0.106 */
 #define INADDR_MAX_LOCAL_GROUP  ((in_addr_t) 0xe00000ff) /* 224.0.0.255 */
 
 #if !__USE_KERNEL_IPV6_DEFS
